src/CLIMA.jl

```diff
@@ -4,6 +4,7 @@ include("Utilities/ParametersType/ParametersType.jl")
 include("Utilities/PlanetParameters/PlanetParameters.jl")
 include("Utilities/RootSolvers/RootSolvers.jl")
 include("Utilities/MoistThermodynamics/MoistThermodynamics.jl")
+include("Utilities/Microphysics/Microphysics.jl")
```

Comment:
should this go under `Atmos`? (same for `MoistThermodynamics`?)

cc @charleskawczynski

Reply:
Yes, `Microphysics` should go under `Atmos/Parameterizations`. Moist thermodynamics is more general (same functions should be used, e.g., for sea ice, snow etc.), so this is better under `Utilities`.